cmd/get/supported-versions.go

```diff
@@ -104,5 +126,9 @@ func FormatSupportedVersions(releases []distribution.KFDRelease, kinds []string)
 		fmtSupportedVersions += "\n"
 	}
 
+	if showUnsupportedFuryctlMsg {
+		fmtSupportedVersions += "* this usually indicates you are not using the latest version of furyctl, try updating or checking the online documentation.\n"
```

Comment:
let's add an empty before the line, otherwise it gets printed right after the table
```suggestion
		fmtSupportedVersions += "\n* this usually indicates you are not using the latest version of furyctl, try updating or checking the online documentation.\n"
```